src/RA_AchievementOverlay.cpp

```diff
@@ -114,7 +114,7 @@ AchievementOverlay::~AchievementOverlay()
     }
 }
 
-void AchievementOverlay::Initialize(HINSTANCE hInst)
+void AchievementOverlay::Initialize(_UNUSED HINSTANCE hInst)
```

Comment:
I'm not familiar with this function, but my first thought is that this is hiding the warning rather than solving the issue it is pointing at.

If there is a legitimate reason to conceal the warning, then please document it here.

Is there no native warning suppression system?

Reply:
LGTM then.